src/components/Tables/Table.tsx

```diff
@@ -48,7 +48,8 @@ const dataToRows = (data: Datum[], headers: Headers): Row[] => {
         const row: Row = { rowId: currentIndex };
 
         for (const headerKey of Object.keys(headers)) {
-            row[headerKey] = datum[headerKey] ?? "";
+            const databaseValue = datum[headerKey] ?? "";
+            row[headerKey] = Array.isArray(databaseValue) ? databaseValue.join() : databaseValue;
```

Comment:
Should the `.join()` function be replaced with `.join(", ")`? Might look a bit nicer with a space between list elements, but it may not matter too much